docs/auth.md

```diff
@@ -146,9 +147,84 @@ the format for the secret changes we will need to create a separate secret for
 just the service catalog today OR we need to add a new `UserServiceAdapter` that
 understands that secret.
 
+### Bearer Auth
+The below section will focus on the bearer token auth.
+
+#### Configuration
+By default, if no authentication is specified the broker will use bearer token auth. The bearer token authentication will use delegated auth from the [kubernetes apiserver](https://github.com/kubernetes/apiserver) library.
+
+The configuration is to grant access, through [kubernetes RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) roles and role-bindings, to the url prefix. The broker has added a configuration option `cluster_url` to specify the url_prefx. This value will default to `ansible-service-broker`. 
+
+Example cluster role:
+```yaml
+- apiVersion: authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: access-asb-role
+  rules:
+  - nonResourceURLs: ["/ansible-service-broker", "/ansible-service-broker/*"]
+    verbs: ["get", "post", "put", "patch", "delete"]
+```
+
+#### Deployment template and secrets
+Here is an example of creating a secret that the service catalog can use, from the above role that will grant access. From the [deployment template](https://github.com/openshift/ansible-service-broker/blob/master/templates/deploy-ansible-service-broker.template.yaml#L224-L248):
```

Comment:
I'm having a hard time parsing the first sentence.

Here is an example of creating a secret that the service catalog can use, from the above role that will grant access.

the last part feels confusing: from the above role that will grant access

See if you can reword that a little more clearly. Right now I'm struggling coming up with a better way to phrase it though.